docs/concepts/portal/dashboard.md

```diff
@@ -0,0 +1,14 @@
+---
+title: " Dashboard Page Reference"
+linkTitle: "Dashboard Page Reference"
+---
+
+
+The Dashboard gives you an unified and summarised view of the application you have deployed. Top view all of your application scroll the Dashboard page. If there are any persistent volumes in error this will be indicated on the dashboard as a red side line next to the application name and will also give you the number of affected volumes 
```

Comment:
`Top view all of your application scroll the Dashboard page.`

Don't think this sentence is needed. Also little typo.